.github/workflows/deploy-docs.yml

```diff
@@ -0,0 +1,38 @@
+name: Build and Deploy docs
+
+on: [push, pull_request]
+
+jobs:
+  build-docs:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Setup Conda
+      uses: s-weigand/setup-conda@v1
+      with:
+        activate-conda: false
+        conda-channels: conda-forge
+
+    - name: Build environment
+      shell: bash -l {0}
+      run: |
+         conda create --name OHW-site --file .binder/conda-linux-64.lock
+         source activate OHW-site
+         conda install rb-bundler compilers
+         bundle install
+    - name: Build documentation
+      shell: bash -l {0}
+      run: |
+        source activate OHW-site
+        bundle exec jekyll build --verbose
+
+    - name: Deploy
+      uses: JamesIves/github-pages-deploy-action@3.7.1
+      if: github.ref == 'refs/heads/source' && github.repository == 'oceanhackweek/oceanhackweek.github.io'
+      with:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        BRANCH: gh-pages
```

Comment:
Change the branch to `master`; that's how the repo is currently set up.

Reply:
Github is showing the default branch with the most recent commits as 
<img width="812" alt="image" src="https://user-images.githubusercontent.com/1296209/121582142-a90bda80-c9fc-11eb-8a34-38bf6ac89d38.png">

Reply:
Yeah, `source` is the content (well, source) branch we commit to. `master` (not `gh-pages`) is the branch that serves the static web site

Reply:
I've pushed a commit to make this change (gh-pages > master)